src/fides/lib/models/client.py

```diff
@@ -36,7 +38,8 @@ def __tablename__(self) -> str:
 
     hashed_secret = Column(String, nullable=False)
     salt = Column(String, nullable=False)
-    scopes = Column(ARRAY(String), nullable=False, default="{}")
+    scopes = Column(ARRAY(String), nullable=True, server_default="{}", default=dict)
+    roles = Column(ARRAY(String), nullable=True, server_default="{}", default=dict)
```

Comment:
Adding new roles to ClientDetail (similar to addition on UserPermissions) and changing the default value because the existing does not work as intended.

One question to clear up. I am allowing multiple "roles" to be assigned to the user for future flexibility, but does product prefer this to be limited to one role?  Originally, the requirements had an 1) Admin 2) Viewer and 3) Privacy Request Manager, but now there's 1) Admin 2) Viewer 3) Privacy Request Manager and 4) Privacy Request Manager + Viewer.   This leads me to believe the intent is now to be assigned just the one "Privacy Request Manager + Viewer" instead of two "Privacy Request Manager" AND "Viewer".

However, if there's ever the possibility that we want multiple roles in the future, I think this should remain a list.